src/main/kotlin/com/example/ticketing_service/queue/infra/RedisQueueListener.kt

```
@@ -0,0 +1,35 @@
+
+        if (emitter != null) {
+            try {
+                // 사용자에게 알림 전송
+                emitter.send(
+                    SseEmitter.event()
+                        .name("entry") // 이벤트 이름
+                        .data("입장 가능") // 데이터
+                )
+                // 알림 보냈으니 연결 종료
+                emitter.complete()
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)